java/component/keyple-plugin/remote/local/src/main/java/org/eclipse/keyple/plugin/remote/impl/LocalServiceClientFactory.java

```diff
@@ -55,6 +57,26 @@ public static NodeStep builder() {
     LocalServiceClient getService();
   }
 
+  public interface NameStep {
+    /**
+     * Configures the service with a specific name.
+     *
+     * @param serviceName identifier of the local service
+     * @return next configuration step
+     * @since 1.0
+     */
+    NodeStep withServiceName(String serviceName);
+
+    /**
+     * Configures the service with the a specific service name. Note that if the service already
+     * exists, it will be overridden
+     *
+     * @return next configuration step
+     * @since 1.0
```

Comment:
Il faut rajouter dans la javadoc un @throws IllegalArgumentException If a service already exists with the default name.